java/client/src/main/java/glide/api/models/ClusterValue.java

```diff
@@ -19,7 +26,7 @@ private ClusterValue() {}
 
     /**
      * Get per-node value.<br>
-     * Check with {@link #hasMultiData()} prior to accessing the data.
+     * Asserts if {@link #hasMultiData()} is false.
```

Comment:
So it would raise an assertion error? I would expect it to throw some client error, the user shouldn't know he needs to also expect assertion errors in addition to the client errors